tools/make-release-packages.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# This script builds and packages a Vitess release suitable for creating a new
+# release on https://github.com/vitessio/vitess/releases.
+
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -euo pipefail
+
+# sudo gem install --no-ri --no-rdoc fpm
+
+source build.env
+
+SHORT_REV="$(git rev-parse --short HEAD)"
+VERSION="5.0.0" # tmp
+# TODO: We can discover version from the last tag, we just need to have this setup.
+# TAG_VERSION="$(git describe --tags --dirty --always | sed  s/v//)"
+
+RELEASE_ID="vitess-${VERSION}-${SHORT_REV}"
+RELEASE_DIR="${VTROOT}/releases/${RELEASE_ID}"
+DESCRIPTION='A database clustering system for horizontal scaling of MySQL
+
+Vitess is a database solution for deploying, scaling and managing large
+clusters of MySQL instances. It’s architected to run as effectively in a public
```

Comment:
shellcheck doesn't like the `'` in this line:
`                               ^-- SC1112: This is a unicode quote. Delete and retype it (or ignore/doublequote for literal).`
We can just double quote the whole `DESCRIPTION`